music21/musicxml/m21ToXml.py

```diff
@@ -6269,8 +6269,6 @@ def testLowVoiceNumbers(self):
         n = note.Note()
         v1 = stream.Voice([n])
         m = stream.Measure([v1])
-        xmlOut = self.getXml(m)
-        self.assertIn('<voice>1</voice>', xmlOut)
```

Comment:
Maybe better to change to `self.assertNotIn()` since we want to test that unnecessary voices get flattened.